src/GitTools.IssueTrackers/IssueTrackers/IssueTrackerBase.cs

```diff
@@ -1,18 +0,0 @@
-﻿namespace GitTools.IssueTrackers
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Threading.Tasks;
-
-    public abstract class IssueTrackerBase : IIssueTracker
```

Comment:
I think adding a base class encourages not thinking about the best way to approach the problem. Most of the time composition is a better idea, if inheritance is the best way then we can add a base class later